### DISCUSSION TITLE
Way to test email notification function?

### DISCUSSION THREAD

**Discussion by tomelsj** [chatwoot/chatwoot#7508] [Self hosted]
Hello, I am struggling to get email notifications to work with self-hosted in Docker, is it a way to test this function, ideally by the rails console (in https://www.chatwoot.com/docs/self-hosted/deployment/docker)? The only way to get errors appears to be in the "Sidekiq monitor", it is not logged in the Docker logs. I am not sure how email notifications are sent in the website chat, is there a delay in how emails are sent? It email notifications only send for example every 10 minutes?`

If someone has a good way to "debug" email notifications, please tell.

**Comment by tayyebi** [comment]
Same here.
